pallets/multisig/src/lib.rs

```diff
@@ -129,7 +129,7 @@ decl_storage! {
         pub MultiSigSignsRequired get(fn ms_signs_required): map hasher(twox_64_concat) T::AccountId => u64;
         /// Number of transactions proposed in a multisig. Used as tx id; starts from 0.
         pub MultiSigTxDone get(fn ms_tx_done): map hasher(twox_64_concat) T::AccountId => u64;
-        /// Proposals presented for voting to a multisig (multisig, proposal id) => Option<proposal>.
+        /// Proposals presented for voting to a multisig (multisig, proposal id) => Option<T::Proposal>.
```

Comment:
Could you explain this typing without copy-pasting the code?

Reply:
Sorry, I don't get it what you want to ask?

Reply:
I meant that copying `(multisig, proposal id) => Option<T::Proposal>` would look cryptic in the docs.